site2/docs/about.md

```diff
@@ -55,7 +55,7 @@ You’ll notice an Edit button at the bottom and top of each page. Click it to o
 
 :::tip
 
-For how to make contributions to documentation, see [Pulsar Documentation Contribution Guide](https://docs.google.com/document/d/11DTnNPpvcPrebLkMAFcDEIFlD8ARD-k6F-LXoIwdD9Y/edit#).
+For how to make contributions to documentation, see [Pulsar Documentation Contribution Guide](../README.md).
```

Comment:
We don't move this `README.md` file to the site repo.

You can use https://github.com/apache/pulsar/blob/master/site2/README.md as a workaround, which I hope we can move into another doc page and update these links to the new page.

Reply:
Sorry I don't fully understand this comment

Now the `doc contribution guide overview` is in `site2/README.md`, and `[Pulsar Documentation Contribution Guide](../README.md)` redirects users to the correct place.

<img width="830" alt="image" src="https://user-images.githubusercontent.com/50226895/196633278-31d618e0-2857-40fa-a764-469613f37c17.png">

Reply:
It works on the GitHub repository but not for the Pulsar website. You can try to preview it and verify if you can jump to the `README.md` file.